docs/en/latest/deployments/gke.md

```diff
@@ -47,7 +47,7 @@ helm repo update
 #  We use Apisix 3.0 in this example. If you're using Apisix v2.x, please set to v2
 ADMIN_API_VERSION=v3
 helm install apisix apisix/apisix \
-  --set gateway.type=LoadBalancer \
+  --set service.type=LoadBalancer \
```

Comment:
I do not find  value `service.type` for apisix. only found `gateway.type` :https://github.com/apache/apisix-helm-chart/blob/24eee91ecd7e4b26c0b3018b1f7c4ff91b53d628/charts/apisix/values.yaml#L223

Reply:
There are two branches. The one you see on docs is from the dev branch where you have `service` https://github.com/apache/apisix-helm-chart/blob/8707bd64d8c881769d93b90133137ad4707c8db7/charts/apisix/values.yaml#L162

When you do helm install, it references the dev branch